content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+For monitoring purposes, it is absurd to use any timezone observing
+so-called daylight saving time. To avoid any kind of timezone
+confusion, we decided to exclusively use UTC for Prometheus. For pure
```

Comment:
I think UTC is less confusing than "Unix time". What the user sees is simply UTC, not Unix time, even if some (but not all) of the interfaces below speak in Unix timestamps.